client/components/Editor/views/TableView.ts

```diff
@@ -0,0 +1,48 @@
+// @ts-expect-error (TableView is exported by this package but not present in its .d.ts definitions)
```

Comment:
This was moved from `plugins/table.ts` to match our newer convention of putting `NodeView` classes in this directory.